src/basic/log.c

```diff
@@ -65,6 +65,7 @@ static int show_color = -1; /* tristate */
 static bool show_location = false;
 static bool show_time = false;
 static bool show_tid = false;
+static bool show_executable = false;
```

Comment:
one of those days we should probably turn these four into a single bitmask value, and also make them settable that way